Trakt/Configuration/configPage.html

```diff
@@ -1,4 +1,4 @@
-﻿<!DOCTYPE html>
+<!DOCTYPE html>
```

Comment:
does this add or remove the BOM?

Reply:
Removed BOM - change not intended. Reverted it back.